composer.json

```diff
@@ -18,7 +18,9 @@
   "require": {
     "bower-asset/colorbox": "^1.6",
     "bower-asset/dropzone": "^5.1",
+    "bower-asset/jquery-simple-color": "^1.2.2",
```

Comment:
Lets try to remove Jquery dependencies and use HTML5 color picker. Please evaluate.